projects/core/src/lib/components/header/header.component.html

```diff
@@ -0,0 +1,45 @@
+<header class="jsmu-header" [ngClass]="{mobile: sideBar}">
+	<nav class="jsmu-header__navigation">
+		<jsmu-avatar [avatarPath]="avatarPath"></jsmu-avatar>
+
+		<div class="jsmu-header__menu-wrapper">
+			<menu class="jsmu-header__menu">
+				<li
+					class="jsmu-header__menu-item jsmu-header__list-item"
+					*ngFor="let menuItem of menuItems"
+				>
+					<a
+						class="jsmu-header__menu-link jsmu-header__link"
+						[routerLink]="menuItem.routerLink"
+						[routerLinkActiveOptions]="{exact:true}"
+						routerLinkActive="jsmu-header__active-link"
+					>
+						{{menuItem.itemName}}
+					</a>
+				</li>
+			</menu>
+
+			<ul class="jsmu-header__authorization">
+				<li class="jsmu-header__logout-item" *ngIf="isLoggedIn; else loggedOut">
+					<jsmu-sign-out></jsmu-sign-out>
+				</li>
+
+				<ng-template #loggedOut>
+					<li class="jsmu-header__login-item">
+						<jsmu-sign-in></jsmu-sign-in>
+					</li>
+				</ng-template>
+			</ul>
+		</div>
+
+		<div
+			class="jsmu-header__burger-menu"
+			[ngClass]="{active: sideBar}"
+			(click)="sideBar=!sideBar"
+		>
+			<span class="jsmu-header__span"></span>
+			<span class="jsmu-header__span"></span>
+			<span class="jsmu-header__span"></span>
```

Comment:
you can use structural directive for this for more convenient record.

Reply:
done